collection-scripts/gather_namespaced_resources

```diff
@@ -87,6 +87,7 @@ oc_get_all+=("volumegroupsnapshots.groupsnapshot.storage.k8s.io")
 oc_get_all+=("volumegroupsnapshots.groupsnapshot.storage.openshift.io")
 oc_get_all+=("obc")
 oc_get_all+=("volumereplication")
+oc_get_all+=("volumegroupreplication")
```

Comment:
Will this fetch the `-oyaml` of all the VGRs present across all the namespaces?

Reply:
yes, oc adm commands collects the yaml